src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2165,6 +2167,7 @@ struct HeatBalanceData : BaseGlobalStruct
     Array1D<Real64> EnclRadThermAbsMult; // EnclRadThermAbsMult  - MULTIPLIER TO COMPUTE 'ITABSF'
     Array1D<bool> EnclSolAbsFirstCalc;   // for error message
     Array1D<bool> EnclRadReCalc;         // Enclosure solar or thermal radiation properties needs to be recalc due to window/shading status change
+    Array1D<bool> no_ht_EMPD_or_HAMT;    // True if surface heat balance uses EMPD or HAMT
```

Comment:
Instead of a new `Array1D` why not just add this as a field in Zone? And maybe name it `noEMPTorHAMT`?